CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing Guidelines
+Brief guidelines to provide a quickstart when contributing. 
+
+This repository is inspired by iproute2 on Linux. More information on the `ip` command can be found in the [man-page](https://man7.org/linux/man-pages/man8/ip.8.html) and [iproute2-page](http://www.policyrouting.org/iproute2.doc.html).
+
+## Developer Setup
+To test your code it can be useful to create an alias with the following command in your terminal:
+```bash
+alias ip="<path-to-repo>/iproute2mac/src/ip.py"
+```
+To have it permanently add the above command to your `~/.zshrc` or `~/.bashrc` file.
+
+Changes should now immediatly take effect. 
```

Comment:
Please leave out the last sentence, as written it could be confusing - Adding the alias to e.g. `.zshrc` will be effective only when opening up a new terminal. To make it effective immediately, one has to either run the command as you are saying above or run e.g. `source ~/.zshrc`.